diff -ruN sogo3.orig/Makefile sogo3/Makefile
--- sogo3.orig/Makefile	2016-07-13 21:54:24.000000000 +0800
+++ sogo3/Makefile	2016-07-15 01:16:45.906083956 +0800
@@ -2,7 +2,7 @@
 # $FreeBSD: head/www/sogo3/Makefile 418484 2016-07-13 13:54:24Z amdmi3 $
 
 PORTNAME=		sogo3
-PORTVERSION=		3.1.3
+PORTVERSION=		3.1.4
 CATEGORIES=		www gnustep
 MASTER_SITES=		http://www.sogo.nu/files/downloads/SOGo/Sources/
 DISTNAME=		SOGo-${PORTVERSION}
diff -ruN sogo3.orig/distinfo sogo3/distinfo
--- sogo3.orig/distinfo	2016-07-13 21:54:24.000000000 +0800
+++ sogo3/distinfo	2016-07-15 01:17:30.374081482 +0800
@@ -1,3 +1,3 @@
-TIMESTAMP = 1468363545
-SHA256 (SOGo-3.1.3.tar.gz) = 57a3521238eca299b655fd2e02e8dae6e968e5dcc5e37461eac27ecc3fb5f624
-SIZE (SOGo-3.1.3.tar.gz) = 32034711
+TIMESTAMP = 1468516650
+SHA256 (SOGo-3.1.4.tar.gz) = 3fd4fa3b078886866c57df03c22558b0329adeac12aef3e93ff022b951f6089c
+SIZE (SOGo-3.1.4.tar.gz) = 32204142
diff -ruN sogo3.orig/pkg-plist sogo3/pkg-plist
--- sogo3.orig/pkg-plist	2016-07-13 21:54:24.000000000 +0800
+++ sogo3/pkg-plist	2016-07-15 01:31:50.513036266 +0800
@@ -1538,6 +1538,7 @@
 GNUstep/Local/Library/SOGo/WebServerResources/js/Common/sgCheckmark.directive.js
 GNUstep/Local/Library/SOGo/WebServerResources/js/Common/sgColorPicker.directive.js
 GNUstep/Local/Library/SOGo/WebServerResources/js/Common/sgCompile.directive.js
+GNUstep/Local/Library/SOGo/WebServerResources/js/Common/sgDatepickerReadonlyInput.directive.js
 GNUstep/Local/Library/SOGo/WebServerResources/js/Common/sgEnter.directive.js
 GNUstep/Local/Library/SOGo/WebServerResources/js/Common/sgEscape.directive.js
 GNUstep/Local/Library/SOGo/WebServerResources/js/Common/sgFocus.directive.js
