diff -ruN sope.orig/Makefile sope/Makefile
--- sope.orig/Makefile	2016-01-20 21:18:59.000000000 +0800
+++ sope/Makefile	2016-02-06 14:12:50.274595167 +0800
@@ -2,7 +2,7 @@
 # $FreeBSD: head/devel/sope/Makefile 406749 2016-01-20 13:18:59Z matthew $
 
 PORTNAME=		sope
-PORTVERSION=		2.3.6
+PORTVERSION=		2.3.8
 CATEGORIES=		devel gnustep
 MASTER_SITES=		http://www.sogo.nu/files/downloads/SOGo/Sources/
 DISTNAME=		SOPE-${PORTVERSION}
diff -ruN sope.orig/distinfo sope/distinfo
--- sope.orig/distinfo	2016-01-20 21:18:59.000000000 +0800
+++ sope/distinfo	2016-02-06 14:51:50.139433317 +0800
@@ -1,2 +1,2 @@
-SHA256 (SOPE-2.3.6.tar.gz) = e8fa039b3b42bbebefc707732aac8fd62478c068c7bc2bd7051d788dea966618
-SIZE (SOPE-2.3.6.tar.gz) = 2275009
+SHA256 (SOPE-2.3.8.tar.gz) = 6fded26f80748bd9628d3c0e54ed44e7689b4df13a596c33a192a008502cd0c3
+SIZE (SOPE-2.3.8.tar.gz) = 2275132
