diff -ruN sope2.orig/Makefile sope2/Makefile
--- sope2.orig/Makefile	2016-06-11 03:44:08.000000000 +0800
+++ sope2/Makefile	2016-07-12 23:03:03.316544978 +0800
@@ -2,7 +2,7 @@
 # $FreeBSD: head/devel/sope2/Makefile 416691 2016-06-10 19:44:08Z acm $
 
 PORTNAME=		sope2
-PORTVERSION=		2.3.11
+PORTVERSION=		2.3.13
 CATEGORIES=		devel gnustep
 MASTER_SITES=		http://www.sogo.nu/files/downloads/SOGo/Sources/
 DISTNAME=		SOPE-${PORTVERSION}
Files sope2.orig/SOPE-2.3.13.tar.gz and sope2/SOPE-2.3.13.tar.gz differ
diff -ruN sope2.orig/distinfo sope2/distinfo
--- sope2.orig/distinfo	2016-06-11 03:44:08.000000000 +0800
+++ sope2/distinfo	2016-07-12 23:55:40.260328027 +0800
@@ -1,3 +1,3 @@
-TIMESTAMP = 1464621616
-SHA256 (SOPE-2.3.11.tar.gz) = 6fb36b5a1ef20c7bf83edad35cd7dc8da5e8b20f6817489e06be18623a686b7b
-SIZE (SOPE-2.3.11.tar.gz) = 2276343
+TIMESTAMP = 1468338940
+SHA256 (SOPE-2.3.13.tar.gz) = 6b1f3492af1e33a9d1eea818a297408deadc6559713eda71c08e54e62c3f376d
+SIZE (SOPE-2.3.13.tar.gz) = 2278988
