diff -ruN rtpproxy.orig/Makefile rtpproxy/Makefile
--- rtpproxy.orig/Makefile	2015-09-22 07:58:34.000000000 +0800
+++ rtpproxy/Makefile	2015-09-22 08:29:58.373724473 +0800
@@ -3,6 +3,7 @@
 
 PORTNAME=	rtpproxy
 PORTVERSION=	2.0.0
+PORTREVISION=	1
 DISTVERSIONPREFIX=	v
 CATEGORIES=	net
 
diff -ruN rtpproxy.orig/files/patch-scripts_rtpproxy.in.freebsd rtpproxy/files/patch-scripts_rtpproxy.in.freebsd
--- rtpproxy.orig/files/patch-scripts_rtpproxy.in.freebsd	2015-09-22 08:19:50.462770410 +0800
+++ rtpproxy/files/patch-scripts_rtpproxy.in.freebsd	2015-09-22 08:15:07.008788976 +0800
@@ -1,15 +1,25 @@
---- scripts/rtpproxy.in.freebsd.orig	2015-03-09 17:40:27 UTC
-+++ scripts/rtpproxy.in.freebsd
-@@ -22,7 +22,10 @@ load_rc_config ${name}
+--- scripts/rtpproxy.in.freebsd.orig	2015-09-22 08:08:06.105815204 +0800
++++ scripts/rtpproxy.in.freebsd	2015-09-22 08:13:54.138790215 +0800
+@@ -6,7 +6,7 @@
+ 
+ # PROVIDE: rtpproxy
+ # REQUIRE: DAEMON
+-# BEFORE: ser openser
++# BEFORE: kamailio opensips
+ 
+ prefix=%%PREFIX%%
+ 
+@@ -22,7 +22,12 @@
  
  rtpproxy_enable=${rtpproxy_enable:-"NO"}
  rtpproxy_laddr=${rtpproxy_laddr:-"0.0.0.0"}
++rtpproxy_ctrl_socket=${rtpproxy_ctrl_socket:-"udp:0.0.0.0:7722"}
++rtpproxy_paddr=${rtpproxy_paddr:-"0.0.0.0"}
 +rtpproxy_usr=${rtpproxy_usr:-"rtpproxy"}
 +rtpproxy_grp=${rtpproxy_grp:-"rtpproxy"}
  
 -command_args="-l ${rtpproxy_laddr} -p /var/run/rtpproxy.pid"
-+command_args="-l ${rtpproxy_laddr} -p /var/run/rtpproxy.pid \
-+  -u ${rtpproxy_usr}:${rtpproxy_grp}"
++command_args="-u ${rtpproxy_usr}:${rtpproxy_grp} -A ${rtpproxy_paddr} -F -l ${rtpproxy_laddr} \
++  -s ${rtpproxy_ctrl_socket} -d INFO -p /var/run/rtpproxy.pid"
  
  run_rc_command "${1}"
-
